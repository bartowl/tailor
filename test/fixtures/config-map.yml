apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    app: bar
  annotations: {}
  name: bar
data:
  bar: baz
