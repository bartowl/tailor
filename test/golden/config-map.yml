apiVersion: v1
data:
  bar: baz
kind: ConfigMap
metadata:
  annotations: {}
  labels:
    app: bar
  name: bar
